### PR TITLE
Fixed login to docker registry with empty email

### PR DESCRIPTION
**Pull request by meetmatt**
Hackery fix

### CODE REVIEW

**Comment by meetmatt** [comment]
Fixed tests

**Comment by bradrydzewski** [comment]
lgtm

File: main.go [hunk 1]
```diff
@@ -144,7 +144,12 @@ func main() {
 
 	// Login to Docker
 	if len(vargs.Username) != 0 {
-		cmd := exec.Command("/usr/bin/docker", "login", "-u", vargs.Username, "-p", vargs.Password, "-e", vargs.Email, vargs.Registry)
+		var cmd *exec.Cmd
+		if len(vargs.Email) == 0 {
```
Comment (tboerger) [inline main.go]: If vars.Email == ""

**Comment by tboerger** [comment]
Maybe we should add that also to master?

**Comment by tboerger** [comment]
Scratch the last comment, master already got it

File: main.go (hunk 1 above)
Comment (meetmatt) [inline main.go]: @tboerger done

**Pull request merged**